firebase/signout.ts

```diff
@@ -0,0 +1,12 @@
+import {getAuth, signOut} from "firebase/auth";
+import firebase_app from "@/firebase/config";
+
+const auth = getAuth(firebase_app);
+
+export default async function signOutUser() {
+    try {
+        return await signOut(auth);
+    } catch (e) {
+        return e;
+    }
+}
```

Comment:
Zou je kunnen uitleggen waarom hier een try catch nodig is als je toch gewoon returned? Kan je dan niet net zo goed return await signOut doen (maar dan is in principe deze hele file niet nodig)?